packages/shared/lib/contexts/governance/utils/isSelectedAccountVoting.ts

```diff
@@ -0,0 +1,7 @@
+import { get } from 'svelte/store'
+import { DEFAULT_PARTICIPATION_OVERVIEW } from '../constants'
+import { participationOverview } from '../stores'
+
+export function isSelectedAccountVoting(): boolean {
+    return get(participationOverview) === DEFAULT_PARTICIPATION_OVERVIEW
```

Comment:
Why would this comparison indicate `isSelectedAccountVoting`?

Reply:
My bad, this is wrong! I fixed it. :sweat_smile: